src/img/icon/manifest.json

```diff
@@ -2,12 +2,12 @@
 	"name": "Pollistics",
 	"icons": [
 		{
-			"src": "\/src\/img\/icon\/android-chrome-192x192.png",
+			"src": "https:\/\/style.pollistics.com\/src\/img\/icon\/android-chrome-192x192.png",
```

Comment:
This was a mistake initially, trying to "install to homescreen" on Android wouldn't have shown the image. Now it does